pyro/infer/mcmc/adaptation.py

```diff
@@ -28,7 +28,7 @@ def __init__(self,
         self.adapt_mass_matrix = adapt_mass_matrix
         self.target_accept_prob = target_accept_prob
         self.is_diag_mass = is_diag_mass
-        self.step_size = 1 if step_size is None else step_size
+        self._init_step_size = 1 if step_size is None else step_size
```

Comment:
Was this additional adaptation related refactoring required for sequential chaining, or does this pertain to some other issue?

Reply:
yup, this is to reset step_size in adapter when we configure it again.